docs/stories/demo.md

```diff
@@ -0,0 +1,85 @@
+# 10101 Demo
+
+We decided to publish a separate demo post so [the pitch](https://makers.bolt.fun/story/10101-pitch--441) can stay concise and focus on being a pitch 😉
+In this demo post that showcases what we have built and how we got there so you get a better idea of how we work and our values.
```

Comment:
Sentence reads a bit weird. Are you trying to pack all `what we have built`, `who we are` and `our values` into one sentence?

Reply:
> Are you trying to pack all what we have built, who we are and our values into one sentence?

Note really :D - I just re-worded the sentence three times and that did not go well :)